software/docker/kafka-can-log/missing

```diff
@@ -0,0 +1,215 @@
+#! /bin/sh
```

Comment:
I don't think this is supposed to be committed. We should add/update `.gitignore`